### PR TITLE
Regen addons

### PR DESCRIPTION
**Pull request by howardjohn**
This is a manual script (since its not deterministic) which i forgot to
run when I updated the dashboard.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by douglas-reid** [comment]
Looks good. Strange error about the large annotations, however.

**Comment by howardjohn** [comment]
@douglas-reid we may have just passed the size limit, ill look into it

File: manifests/addons/dashboards/istio-performance-dashboard.json [hunk 1]
```diff
@@ -97,14 +97,6 @@
       "stack": false,
       "steppedLine": false,
       "targets": [
-        {
```
Comment (howardjohn) [inline manifests/addons/dashboards/istio-performance-dashboard.json:100]: These were made in a previous change, but made directly to the generated output

File: manifests/addons/gen.sh [hunk 2]
```diff
@@ -46,24 +46,33 @@ helm3 template prometheus stable/prometheus \
...
 # Set up grafana
 {
   helm3 template grafana stable/grafana \
     --namespace istio-system \
     --version 5.3.5 \
     -f "${WD}/values-grafana.yaml"
 
-  # Set up grafana dashboards. Split into 2 to avoid Kubernetes size limits
+  # Set up grafana dashboards. Split into 2 and compress to single line json to avoid Kubernetes size limits
+  compressDashboard "pilot-dashboard.json"
```
Comment (howardjohn) [inline manifests/addons/gen.sh:61]: Added compression to fit in k8s. This cuts the size by about 40% giving us some breathing room

**Review by sdake — APPROVED** [review]
LGTM from ENV perspective. I don't know if the dashboard changes are correct or not, hence will need a review from telemetry.

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**